packages/@remirror/extension-events/src/events-extension.ts

```diff
@@ -14,6 +14,20 @@ export interface EventsOptions {
    * Return `true` to prevent any other prosemirror listeners from firing.
    */
   focus: Handler<(event: FocusEvent) => boolean | undefined>;
+
+  /**
+   * Listens for mousedown events on the editor.
+   *
+   * Return `true` to prevent any other prosemirror listeners from firing.
+   */
+  mousedown: Handler<(event: MouseEvent) => boolean | undefined>;
```

Comment:
Wasn't sure on `mousedown` or `mouseDown` here.

Reply:
You picked correctly 👍